applications/CompressiblePotentialFlowApplication/tests/wake_process_3d_tests/25_elements_nodes_on_wake_test/wake_stl.stl

```diff
@@ -1,30 +1,30 @@
 solid GiD
-  facet normal               0              -0               1  
+  facet normal               0              -0               1
     outer loop
-      vertex               5               0               5
-      vertex              10               0               5
-      vertex             7.5               5               5
+      vertex               5               0               4.9
```

Comment:
These changes were related to the cutting of wing tips? Is this needed after switching to the DiscDistance process?

Reply:
This was because in this case the nodes are laying exactly on the wake and we changed the behavior of the code. I think it is still necessary, but I will double check

Reply:
You are right! This is no longer needed after changing to the DiscDistance, I must have forgotten to check it, Ill revert these changes, thanks for the review!